IndexationEngine.py

```diff
@@ -0,0 +1,73 @@
+"""
+    Search Engine
+    by Vincent Jeanselme
+    and Matthieu Clin
+    vincent.jeanselme@gmail.com
+
+    Indexation part from the Engine. Here we are indexing all documents in order to make them available trough
+    later search
+"""
+
+import argparse
+import os
+import IndexationFactory
+
+
+class CheckValidPath(argparse.Action):
+    """
+        Checks if each document provided is a valid path for a file
```

Comment:
I do not understand what this function does ?
Could you add a comment, please ?